src/lib/Input/index.js

```diff
@@ -218,6 +233,12 @@ class Input extends React.Component {
       case `password`:
         type = `password`
         break
+      case `ssn`:
```

Comment:
I don't think we need this for ssn and ein since they are not html attributes. https://www.w3schools.com/tags/att_input_type_password.asp